sdk/CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 0.5.0 - YYYY-MM-DD
+
+### Changed
+
+- Rename `Account::signEvm` to `signSecp256k1Ecdsa` and `EvmSignature` to `Secp256k1EcdsaSignature`;
```

Comment:
This is not what I meant, the new bindings have been released and there is now a changelog for it.

Reply:
Ohhh